arelle/plugin/validate/EDINET/rules/gfm.py

```diff
@@ -699,6 +699,35 @@ def rule_gfm_1_3_10(
             )
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[DISCLOSURE_SYSTEM_EDINET],
+)
+def rule_gfm_1_3_11(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    EDINET.EC5700W: [GFM 1.3.11] The usedOn attribute of the extended link role should be set to include all of link:presentationLink, link:calculationLink, and link:definitionLink.
+    """
+    requiredUsedOns = {XbrlConst.qnLinkPresentationLink,
+                       XbrlConst.qnLinkCalculationLink,
+                       XbrlConst.qnLinkDefinitionLink}
+    for modelRoleTypes in val.modelXbrl.roleTypes.values():
+        modelRoleType = modelRoleTypes[0]
```

Comment:
With `roleTypes` being a `defaultdict`, I'm hesitant to assume the list is never empty. I can see at least one place in the codebase where `roleTypes[linkroleUri]` is referenced (and thus a list is created) without anything being added.